server/lyft/checks/github_client.go

```diff
@@ -0,0 +1,41 @@
+package checks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/runatlantis/atlantis/server/events/vcs"
+	"github.com/runatlantis/atlantis/server/events/vcs/types"
+	"github.com/runatlantis/atlantis/server/logging"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
+)
+
+// Wrapping github client with a struct to allow for method resolution
+type GithubClient struct {
```

Comment:
Why do you even need this? Can't you directly assign the struct to `ChecksClientWrapper`?

Reply:
Hmm I was running into some errors because I was trying to assign the client to `vcs.GithubClient`, instead if I just use `GithubClient` it works. So, I've removed this struct and directly assigned the client to the `ChecksClientWrapper`.